frontend/package.json

```diff
@@ -25,7 +25,7 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "lint-fix": "eslint --fix ./"
+    "lint-fix": "eslint --fix ./src/"
```

Comment:
Why was this changed to the src/ directory? It looks like most of the javascript code is under the frontend/ directory